fb_instant_articles.module

```diff
@@ -354,60 +333,161 @@ function fb_instant_articles_fb_instant_articles_transformer_rules() {
+          'selector' => 'div.embed',
+        ),
+        'interactive.height' => array(
+          'type' => 'int',
+          'selector' => 'iframe',
+          'attribute' => 'height',
+        ),
+        'interactive.width' => array(
+          'type' => 'int',
+          'selector' => 'iframe',
+          'attribute' => 'width',
         ),
       ),
     ),
     array(
-      'class' => 'SocialEmbedRule',
-      'selector' => '//p[iframe]',
+      'class' => 'InteractiveRule',
+      'selector' => 'div.interactive',
       'properties' => array(
-        'socialembed.url' => array(
+        'interactive.iframe' => array(
+          'type' => 'children',
+          'selector' => 'div.interactive',
+        ),
+        'interactive.height' => array(
+          'type' => 'int',
+          'selector' => 'iframe',
+          'attribute' => 'height',
+        ),
+        'interactive.width' => array(
+          'type' => 'int',
+          'selector' => 'iframe',
+          'attribute' => 'width',
+        ),
+      ),
+    ),
+    array(
+      'class' => 'InteractiveRule',
+      'selector' => "//div[@class='embed' and iframe]",
+      'properties' => array(
+        'interactive.url' => array(
           'type' => 'string',
           'selector' => 'iframe',
           'attribute' => 'src',
         ),
-        'socialembed.iframe' => array(
+        'interactive.iframe' => array(
           'type' => 'children',
           'selector' => 'iframe',
+          'attribute' => 'src',
         ),
-        'socialembed.caption' => array(
-          'type' => 'element',
-          'selector' => 'figcaption',
+        'interactive.width' => array(
+          'type' => 'int',
+          'selector' => 'iframe',
+          'attribute' => 'width',
+        ),
+        'interactive.height' => array(
+          'type' => 'int',
+          'selector' => 'iframe',
+          'attribute' => 'height',
         ),
       ),
     ),
     array(
-      'class' => 'SocialEmbedRule',
-      'selector' => 'div.embed',
+      'class' => 'InteractiveRule',
+      'selector' => "//div[@class='interactive' and iframe]",
       'properties' => array(
-        'socialembed.iframe' => array(
+        'interactive.url' => array(
+          'type' => 'string',
+          'selector' => 'iframe',
+          'attribute' => 'src',
+        ),
+        'interactive.iframe' => array(
           'type' => 'children',
-          'selector' => 'div.embed',
+          'selector' => 'iframe',
+        ),
+        'interactive.height' => array(
+          'type' => 'int',
+          'selector' => 'iframe',
+          'attribute' => 'height',
+        ),
+        'interactive.width' => array(
+          'type' => 'int',
+          'selector' => 'iframe',
+          'attribute' => 'width',
         ),
       ),
     ),
     array(
-      'class' => 'SocialEmbedRule',
-      'selector' => 'div.oembed',
+      'class' => 'InteractiveRule',
+      'selector' => 'table',
```

Comment:
This is a bit of a stretch, any `table`'s match `InteractiveRule`? This also feels like more of a custom rule that you implement with `hook_fb_instant_articles_transformer_rules` in your own custom module.